spec/unit/block_score/company_spec.rb

```diff
@@ -1,80 +1,60 @@
+  describe '.new' do
+    subject(:company) { described_class.new(attributes_for(:company)) }
 
-      it { is_expected.not_to be_persisted }
-      its(:class) { should be BlockScore::Company }
-    end
-
-    describe '.create' do
-      context 'valid company' do
-        subject(:company) { BlockScore::Company.create(attributes_for(:company)) }
-
-        it { is_expected.to be_persisted }
-        its(:class) { should be BlockScore::Company }
-      end
+    it { is_expected.not_to be_persisted }
+    it { is_expected.to be_an_instance_of described_class }
+  end
 
-      context 'invalid company' do
-        subject(:company) { BlockScore::Company.create(attributes_for(:invalid_company)) }
+  describe '.create' do
+    context 'valid company' do
+      subject(:company) { described_class.create(attributes_for(:company)) }
 
-        it { is_expected.to be_persisted }
-        its(:class) { should be BlockScore::Company }
-      end
+      it { is_expected.to be_persisted }
+      it { is_expected.to be_an_instance_of described_class }
     end
 
-    describe '.retrieve' do
-      let(:company_id)  { create(:company).id }
-      subject(:company) { BlockScore::Company.retrieve(company_id) }
+    context 'invalid company' do
+      subject(:company) { described_class.create(attributes_for(:invalid_company)) }
 
       it { is_expected.to be_persisted }
-      its(:entity_name) { is_expected.not_to be_empty }
-      its(:class) { should be BlockScore::Company }
+      it { is_expected.to be_an_instance_of described_class }
     end
+  end
 
-    describe '.all' do
-      let(:uniq_token_one) { '4580ab610751f6f37078329357ff98db' }
-      let(:uniq_token_two) { '0f222bcb487f9de689c5c60b854973bc' }
-      subject(:companies) { BlockScore::Company.all }
-
-      before do
-        create(:company, entity_name: uniq_token_one)
-        create(:company, entity_name: uniq_token_two)
-      end
+  describe '.retrieve' do
+    let(:company_id) { create(:company).id }
+    subject(:company) { described_class.retrieve(company_id) }
 
-      it { is_expected.not_to be_empty }
-      it { expect(companies[0].entity_name).to eq uniq_token_two }
-      it { expect(companies[1].entity_name).to eq uniq_token_one }
-    end
+    it { is_expected.to be_persisted }
+    its(:entity_name) { is_expected.not_to be_empty }
+    it { is_expected.to be_an_instance_of described_class }
+  end
 
-    describe '#refresh' do
-      subject(:company) { create(:company, entity_name: 'BlockScore') }
-      before do
-        company.entity_name = 'DarkScore'
-        company.refresh
-      end
+  describe '.all' do
+    let(:uniq_token_one) { '4580ab610751f6f37078329357ff98db' }
+    let(:uniq_token_two) { '0f222bcb487f9de689c5c60b854973bc' }
+    subject(:companies) { described_class.all }
 
-      its(:entity_name) { is_expected.to eq 'BlockScore' }
+    before do
+      create(:company, entity_name: uniq_token_one)
+      create(:company, entity_name: uniq_token_two)
```

Comment:
Instance of https://github.com/BlockScore/blockscore-ruby/pull/68/files#r52838300